apps/language_server/lib/language_server/providers/completion.ex

```diff
@@ -607,33 +642,48 @@ defmodule ElixirLS.LanguageServer.Providers.Completion do
               Keyword.merge(
                 options,
                 pipe_before?: pipe_before?,
-                capture_before?: capture_before?
+                capture_before?: capture_before?,
+                trigger_signature?: trigger_signature?,
+                locals_without_parens: locals_without_parens,
+                text_after_cursor: text_after_cursor,
+                with_parens?: with_parens?
               )
             )
 
           {label, insert_text}
       end
 
-    detail =
-      cond do
-        spec && spec != "" ->
-          spec
+    detail_header =
+      if inspect(module) == origin do
+        "#{visibility} #{type}"
+      else
+        "#{origin} #{type}"
+      end
 
-        String.starts_with?(type, ["private", "public"]) ->
-          String.replace(type, "_", " ")
+    footer =
+      if String.starts_with?(type, ["private", "public"]) do
+        String.replace(type, "_", " ")
+      else
+        SourceFile.format_spec(spec, line_length: 30)
+      end
 
-        true ->
-          "(#{origin}) #{type}"
+    command =
+      if trigger_signature? do
+        %{
+          "title" => "Trigger Parameter Hint",
+          "command" => "editor.action.triggerParameterHints"
```

Comment:
This command appears to be VSCode-specific. When testing with Emacs it appears to at least not cause any direct problems, but I would like to at least explicitly note that the command is VSCode-specific. For now maybe that can be accomplished by extracting it into a short new module. It would be good if we could get someone to test this on vim.

Reply:
I think we should leave it as it is. Other editors can support that in the future if they want, don't they?

Reply:
Okay, that seems to work. Looks like most clients simply ignore commands that they don't understand.